### ISSUE TITLE
failed with result 'exit-code'

### ISSUE THREAD

**Issue by weplick** [infokek/suricata-telegram-notifier#1]
Hello, I'm just trying to send suricata alerts to telegram bot and I new to this stuff so I get this error
![image](https://github.com/infokek/suricata-telegram-notifier/assets/167795779/2c4c9b25-293b-4395-9172-6a4e5f095283)
so it's running active or it's just says it' loaded
I guess I didn't installed it correctly? I don't know
Also in your "service" folder you have some .py files, should I download and do something with them also? Bcz your instruction is not saying that
I did already installed suricata and set it up, it's working, also added some rules
I'm doing hping3 command for dos attack, so just basically need to send them alerts in /var/suricata/fast.log to telegram bot
Sorry, I'm just new to this so I wanna ask you to provide any solution

**Comment by infokek** [comment]
Hello @weplick! Sorry for long answer. 
Please redo following steps:
1) `git clone https://github.com/infokek/suricata-telegram-notifier.git -b main`
2) `cd suricata-telegram-notifier`
3) `cp configs/service.ini.example configs/service.ini`
4) edit _configs/service.ini_ by your own
5) `chmod +x install.sh`
6) `sudo ./install.sh`

I've just tried on **Ubuntu 22.04.1** and there are no problems. Please let me know if you have any problems. Good luck.

**Comment by weplick** [comment]
> Hello @weplick! Sorry for long answer. Please redo following steps:
> 
> 1. `git clone https://github.com/infokek/suricata-telegram-notifier.git -b main`
> 2. `cd suricata-telegram-notifier`
> 3. `cp configs/service.ini.example configs/service.ini`
> 4. edit _configs/service.ini_ by your own
> 5. `chmod +x install.sh`
> 6. `sudo ./install.sh`
> 
> I've just tried on **Ubuntu 22.04.1** and there are no problems. Please let me know if you have any problems. Good luck.

Yeah, it works now, thanks! I also changed path way in checker.py to /var/log/suricata/fast.log, bcz it seems it didn't catch fast.log properly. So, for now it works, but only once everytime I start the machine. I guess it's how it should work?
